preduzece/src/main/java/raf/si/racunovodstvo/preduzece/controller/ObracunZaradeRestController.java

```diff
@@ -0,0 +1,48 @@
+package raf.si.racunovodstvo.preduzece.controller;
+
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import raf.si.racunovodstvo.preduzece.jobs.ObracunZaradeJob;
+import raf.si.racunovodstvo.preduzece.services.impl.ObracunZaradeService;
+
+import java.util.Date;
+import java.util.Map;
+
+@CrossOrigin
+@RestController
+@SecurityRequirement(name = "bearerAuth")
+@RequestMapping("/api/obracun_zarade")
+public class ObracunZaradeRestController {
+    private final ObracunZaradeService obracunZaradeService;
+    private final ObracunZaradeJob obracunZaradeJob;
+
+    public ObracunZaradeRestController(ObracunZaradeService obracunZaradeService, ObracunZaradeJob obracunZaradeJob) {
+        this.obracunZaradeJob = obracunZaradeJob;
+        this.obracunZaradeService = obracunZaradeService;
+    }
+
+    @PostMapping(value = "/{dan}")
+    public ResponseEntity<?> changeJobDayOfMonth(@PathVariable Integer dan) {
+        obracunZaradeJob.setDayOfMonth(dan);
+        return ResponseEntity.ok().build();
+    }
+
+    @PostMapping()
+    public ResponseEntity<?> setObracunZaradeNaziv(@RequestBody Map<String, String> request) {
+        obracunZaradeService.updateObracunZaradeNaziv(Long.parseLong(request.get("obracunZaradeId")), request.get("naziv"));
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping(value = "/{datum}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<?> getObracunZarade(@PathVariable Date datum) {
+        return ResponseEntity.ok(obracunZaradeService.findByDate(datum));
+    }
```

Comment:
Da li si ovo proverio da li radi?
Mislim da spring po default-u ne moze da parsira date, mora da se postavi neki DateTimeFormatting. Do sada smo koristili ISO_DATE, bilo bi ok da se to koristi i ovde.